pkg/admission/logicalcluster/admission.go

```diff
@@ -191,7 +191,7 @@ func (o *plugin) Validate(ctx context.Context, a admission.Attributes, _ admissi
 			return fmt.Errorf("LogicalCluster cannot be deleted: %w", err)
 		}
 		groups := sets.NewString(a.GetUserInfo().GetGroups()...)
-		if !logicalCluster.Spec.DirectlyDeletable && !groups.Has(kuser.SystemPrivilegedGroup) && !groups.Has(bootstrap.SystemLogicalClusterAdmin) {
+		if !logicalCluster.Spec.DirectlyDeletable && !groups.Has(kuser.SystemPrivilegedGroup) && !groups.Has(bootstrap.SystemLogicalClusterAdmin) && !groups.Has(bootstrap.SystemExternalLogicalClusterAdmin) {
```

Comment:
i may misread: but if the user is external logical cluster admin, it already short-circuits in line 130, no?

Reply:
Hmm yes, I just copied the existing `SystemLogicalClusterAdmin` but it seems the conditional should simply be `if !logicalCluster.Spec.DirectlyDeletable` as the other conditions are redundant - I'll add a commit to clean this up, thanks!

Reply:
Done